### ISSUE TITLE
Voucher total usage times limit is 0 by default

### ISSUE THREAD

**Issue by LeBenLeBen** [mirumee/saleor#4988]
I'm trying to use a voucher code in a checkout but it is not accepted despite fulfilling all the constraints.

The problem comes from the checkbox "Limit number of times this discount can be used in total", which has a value of 0 even when being unchecked. Changing this value to a higher number allows to use the voucher properly.

### Steps to reproduce the problem
1. Go to discounts/vouchers in the dashboard
2. Create a new voucher with just a code and an amount
3. Try to use the voucher in a checkout
-> An error is thrown saying the voucher code doesn't exists

### What I expected to happen
The voucher should be accepted because I didn't check the "Limit number of times this discount can be used in total" checkbox.

**Comment by maarcingebala** [comment]
I agree - it should work as you described it. Do you know if `0` is a default value used in API if there is no value in the input, or is this a default sent by the dashboard app? Anyway we'll take a look at it.

**Comment by LeBenLeBen** [comment]
In the dashboard, I can see the value being "0" by default if I check the checkbox.
